sp/src/game/server/hl2/weapon_physcannon.cpp

```diff
@@ -1042,11 +1046,22 @@ void CPlayerPickupController::Init( CBasePlayer *pPlayer, CBaseEntity *pObject )
 	}
 
 	CHL2_Player *pOwner = (CHL2_Player *)ToBasePlayer( pPlayer );
+#ifndef MAPBASE
 	if ( pOwner )
 	{
 		pOwner->EnableSprint( false );
 	}
 
+#else
+	if ( pOwner && sv_player_enable_propsprint.GetBool() == false )
+	{
+		pOwner->EnableSprint( false );
+	}
+	else
+	{
+		pOwner->EnableSprint( true );	
+	}
+#endif
```

Comment:
`pOwner` cannot be null for either of these statements, so they should be nested within the original check.
```suggestion
	if ( pOwner )
	{
#ifndef MAPBASE
		pOwner->EnableSprint( false );
#else
		if ( sv_player_enable_propsprint.GetBool() == false )
		{
			pOwner->EnableSprint( false );
		}
		else
		{
			pOwner->EnableSprint( true );	
		}
#endif
	}
```